frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -440,6 +594,28 @@ export const ClusterServiceVersionList: React.SFC<ClusterServiceVersionListProps
       EmptyMsg={NoOperatorsMatchFilterMsg}
       NoDataEmptyMsg={NoDataEmptyMsg}
       virtualize
+      customSorts={{
+        managedNamespace: (obj) => {
```

Comment:
Don't duplicate this logic... create a function outside of the component that you reference here

Reply:
+1 great time to deduplicate a couple of things.